addons/snailmail/static/src/js/snailmail_external_layout.js

```diff
@@ -1,14 +0,0 @@
-// @odoo-module ignore
-
-// Change address font-size if needed
-document.addEventListener('DOMContentLoaded', function (evt) {
-    var recipientAddress = document.querySelector(".address.row > div[name='address'] > address");
-    let baseSize = 120;
-    if (!recipientAddress) {
-        recipientAddress = document.querySelector("div .row.fallback_header > div.col-5.offset-7 > div:first-child");
-    }
-    var style = window.getComputedStyle(recipientAddress, null); 
-    var height = parseFloat(style.getPropertyValue('height'));
-    var fontSize = parseFloat(style.getPropertyValue('font-size'));
-    recipientAddress.style.fontSize = (baseSize / (height / fontSize)) + 'px';
-});
```

Comment:
Doesn't this mean we don't handle long addresses anymore, as they won't fit in the dedicated space ?

Reply:
@lobaudoux You are right, but if the size is reduced it will bring the tax id or the phone number in the postable area, failing as well 😞 We still have quite a large amount of horizontal space in the postable area to fit most addresses.  
![image](https://github.com/user-attachments/assets/0a5b841a-bb9a-48f5-94ff-36a7c07feccf)

I see a few solutions
1) Accept as is (can't we set a cover page with raw data in pingen for the unfitting cases?)
2) Let the address line wrap and use more vertical space / reduce line-height to still fit the address area
3) Resize with Js but calculate line-height / spacing accordingly to ensure the tax id (or phone) is not getting back in the postable area

I'm open to more suggestions, I don't really know which is best there and if I'm unaware of all the limitations

Reply:
IMO, it's ok to leave it as it is, as you pointed out there is always the cover page backup in the exceptional case where the address doesn't fit.

Hello @cth-odoo, could we have your opinion on this ?

Reply:
Hello @lobaudoux and @mano-odoo 

Answer 1.